src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateCreationTests.JsonBody.cs

```diff
@@ -122,9 +122,7 @@ public async Task MapAction_ExplicitBodyParam_ComplexReturn_Snapshot()
         };
         var source = $"""
 app.MapPost("/fromBodyRequired", ([FromBody] Todo todo) => TypedResults.Ok(todo));
-#pragma warning disable CS8622
```

Comment:
Interceptors fixed generating overloads for methods with the same signature but differing nullability so we no longer have a warning to suppress.